calculator/src/App.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import "./styles/App.css";
+import Display from "./components/display";
+import Keypad from "./components/keypad";
+
+class App extends React.Component {
+  state = { input: '',
```

Comment:
I think it's better to have your operator in your state than it is to just have result and using eval